var/spack/repos/builtin/packages/flux-sched/package.py

```diff
@@ -186,6 +186,9 @@ def setup_run_environment(self, env):
 
 class CMakeBuilder(CMakeBuilder):
     def cmake_args(self):
+        ver_in_src = os.path.exists(os.path.join(self.stage.source_path, "flux-sched.ver"))
+        if self.spec.satisfies("@:0.33") or not ver_in_src:
```

Comment:
```suggestion
        # flux-sched before v0.33 does not correctly set the version even when the file is present.
        if self.spec.satisfies("@:0.33") or not ver_in_src:
```

No worries, I'd probably suggest documenting the edge case here so that someone doesn't some and change it later on by accident.